unreal_asset/src/types/fname.rs

```diff
@@ -50,6 +50,29 @@ pub enum FName {
     },
 }
 
+impl std::fmt::Debug for FName {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> Result<(), std::fmt::Error> {
+        match self {
+            FName::Backed {
+                index,
+                number,
+                ty,
+                name_map,
+            } => f
+                .debug_struct("Backed")
```

Comment:
This implies there is a struct called `Backed`. Consider changing this to `FName::Backed`. Same for Dummy

Reply:
the derive debug also does this